Resources/Prototypes/Entities/Objects/Power/powercells.yml

```diff
@@ -36,6 +36,15 @@
   components:
   - type: PowerCell
     cellSize: Medium
+  - type: SolutionContainerManager
+    solutions:
+      powerCell:
+        maxVol: 10
+  - type: Extractable
+    juiceSolution:
+      reagents:
+        - ReagentId: Licoxide
+          Quantity: 10
```

Comment:
licoxide shouldn't increase with size like this

Reply:
Done